source/game_sa/Entity/Vehicle/CBoat.cpp

```diff
@@ -120,6 +285,149 @@ void CBoat::AddWakePoint(CVector posn)
         ++m_nNumWaterTrailPoints;
 }
 
+bool CBoat::IsSectorAffectedByWake(CVector2D vecPos, float fOffset, CBoat** ppBoats)
+{
+    if (!CBoat::apFrameWakeGeneratingBoats[0])
+        return false;
+
+    bool bWakeFound = false;
+    for (size_t i = 0; i <= 3; ++i) {
```

Comment:
```suggestion
    for (size_t i = 0; i <= TOTAL_WAKE_POINTS; ++i) {
```
you can define `TOTAL_WAKE_POINTS` in CBoat.h

Reply:
Fixed, bit different name, but same idea :D